### DISCUSSION TITLE
Assistance in the ecosystem of multi-tenants & Panels

### DISCUSSION THREAD

**Discussion by DevMushref** [filamentphp/filament#7499] [Help]
Hello everyone.

I Understood the documentation of the Panels and Tenants but I need help with the ecosystem of the application.

I have a Tenant/Admin called HSSE, within it there are teams:
- HSSE
    * Security
       * Control Center
       * CCTV
       * Access Control
    * Safety
       * Fire Station
       * Ambulance
    * Environment

what should be the right way of setting such structure? Your assistance is highly appreciated.

**Comment by pboivin** [comment]
What are panels and what are resources in this example?

**Reply by DevMushref** [reply]
That's what I'm trying to figure out, which should be a panel. I need the "Security" to be able to manage all that is within it.
The "Control Center" will have its own resources, as well as CCTV & Access Control, each has it's own users, roles & permission. Same Applies for the rest. However, Each team may not access the resources other than itself.

**Reply by pboivin** [reply]
Maybe `Security` can be a panel and `Control Center` can be a navigation group with some resources in it?

**Reply by DevMushref** [reply]
I could do what you recommended. I have a concern:

Is it possible to separate the users of the panels? I don't want the `security` to access `safety` and vice-versa. Should I create different `users` tables per panel?

**Reply by pboivin** [reply]
I don't think different `users` tables is necessary. You can control the access in `canAccessPanel`:
https://filamentphp.com/docs/3.x/panels/users#authorizing-access-to-the-panel
